README.md

```diff
@@ -1,8 +1,27 @@
 # terraform-google-kubernetes-cluster
 GKE Kubernetes module with node pools submodule
 
+Table of contents
+=================
+
+   * [terraform-google-kubernetes-cluster](#terraform-google-kubernetes-cluster)
+      * [1. Features](#1-features)
+      * [2. Usage:](#2-usage)
+         * [a) cluster with default node pool on preemptible](#a-cluster-with-default-node-pool-on-preemptible)
```

Comment:
My proposal is to remove `a, b, c` and just leave unordered list, how about that?

Reply:
👍